src/index.js

```diff
@@ -60,6 +61,18 @@ async function run() {
     reviewers.push(...default_reviwers);
   }
 
+  const DEFAULT_OPTIONS = {
+    numberOfReviewers: 0,
+  };
```

Comment:
This will be unnecessary if you use "nothing is set" as the default value to indicate there should be no limit.

Reply:
Resolved in #30.